Session/Conversations/ConversationMessageMapping.swift

```diff
@@ -229,7 +229,7 @@ public class ConversationMessageMapping: NSObject {
         let indexPtr: UnsafeMutablePointer<UInt> = UnsafeMutablePointer<UInt>.allocate(capacity: 1)
         let wasFound = view.getGroup(nil, index: indexPtr, forKey: uniqueId, inCollection: TSInteraction.collection())
         guard wasFound else {
-            owsFailDebug("Could not find interaction.")
+            SNLog("Could not find interaction.")
```

Comment:
Nice, always wanted to modify this and always forgot...

Reply:
Haha yea, I'd like to properly debug it, understand what is happening and fix the case where it's getting hit but it's been pretty frustrating when debugging message requests because it seems to like to assert on the first message send quite frequently...